README.md

```diff
@@ -0,0 +1,60 @@
+# django-peertube-runner-connector: A django application to connect to a peertube runner and transcode videos 
+
+## Overview 
+django-peertube-runner-connector is designed to use Peertube transcoding runners outside of Peertube App. It implements a set of endpoints and a SocketIO server that allow runners to request jobs, updated job status, download media files and upload the transcoded media files. It provides two endpoints that can be used by users to transcode a given video.
+
+
+## Architecture
+
+The API in itself is separated in three parts.
+
+### Runner API
+
+This part will interact with Peertube runners. It is not designed to be used by a user as it reproduces what the Peertube App is doing in order to manage runners / jobs.
+
+
+#### Runner Behavior
+
+Jobs are stored in a Database, and runners hit the `/request` endpoint to get the available jobs to transcode.
+
+
+### Video API
+
+This API provides endpoints transcode a video. Both endpoints works in the same way: it receives a video file and a name, and then creates transcoding jobs for it. The difference between the two endpoints is that one takes a File and the other takes a path to a file.
+
+We use function `probe` of python-ffmpeg library, to get a thumbnail and all the necessary metadata to creates transcoding jobs. Once the jobs are created, the WebSocket server emits an event to inform runners of a new pending jobs.
+
+The two endpoints available are: 
+
+- `video/upload`: this endpoint takes a File and a name to transcode the given File
+- `video/transcode`: this endpoint takes path and a name. It will then verify if the given the given file can be found a the path (see Storage section to know the path) abd create transcoding jobs for it.
```

Comment:
```suggestion
- `video/transcode`: this endpoint takes a path and a name. It will then verify if the given file can be found at the path (see Storage section to know the path) and create transcoding jobs for it.
```

I'm not sure to understand this sentence. This part: `if the given file can be found at the path`.